torch/quantization/_correct_bias.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function, division, absolute_import
+import torch
+import torch.nn as nn
+import torch.nn.quantized as nnq
+
+import torch.quantization
+import torch.quantization._numeric_suite as ns
+
+_supported_modules = {nn.Linear, nn.Conv2d}
+_supported_modules_quantized = {nnq.Linear, nnq.Conv2d}
+
+def get_module(model, name):
```

Comment:
I think you can call `model.named_modules()` to get a dictionary of name -> submodules and just use that dictionary to get the submodule.